README.md

```diff
@@ -1,23 +1,92 @@
+### Status light
+
+The status light lights up when WRunt has reached the target. This allows observers to know when WRunt knows it has reached its target. During a competition setting, this lets the judges know that the robot actually completed the task and did not just accidentally get to the right spot.
+
+### Continue to next beacon signal
 
-#### Continue to next beacon signal
 Once WRunt has reached the first beacon, you will need to give it a manual signal that it can move on to the next beacon. In real life, this signal would be transmitted over a network connection. For our pursposes, it will be a button on a Graphical User Interface (GUI).
 
 ## WRunt behavior flowchart
+
 1. WRunt determines the closest beacon that hasn't been visited yet
-2. WRunt move to and reaches the target beacon 
+2. WRunt move to and reaches the target beacon
 3. WRunt marks the beacon as visited, changes its own status light, and awaits a certain number of seconds
-4. Repeat steps 1-3 until all beacons have been reached
\ No newline at end of file
+4. Repeat steps 1-3 until all beacons have been reached
+
+## How To Open The Project
+
+This project uses Docker to efficiently containerize the workspace and make cross-platform development easier.  Find your host platform below to see how to start development:
+
+### Linux
+
+#### Tools:
+
+* [git](https://git-scm.com/)
+* [VS Code](https://code.visualstudio.com/)
+* [Docker Engine](https://docs.docker.com/engine/)
+
+#### Instructions:
+
+These instructions are written for Ubuntu, steps may vary if using a different distribution.
+
+1. Install git: `sudo apt install git`
+2. Install VS Code: https://code.visualstudio.com/docs/setup/linux
+3. Install Docker: https://docs.docker.com/engine/install/ubuntu/
+4. Set up SSH keys
+5. Clone the repository
+6. Open the repository with VSCode
+7. Reopen the folder in container
+
+#### Troubleshooting:
+
+### Windows
+
+#### Tools:
+
+* [git](https://git-scm.com/)
+* [VS Code](https://code.visualstudio.com/)
+* [WSL 2](https://www.omgubuntu.co.uk/how-to-install-wsl2-on-windows-10)
+* [Docker Desktop](https://www.docker.com/products/docker-desktop/)
+
+#### Instructions:
+
+1. Install tools
+2. Set up SSH keys on Windows and in WSL
+3. Clone the repository inside WSL
+4. Open the repository with VSCode
+5. Launch Docker
+6. Reopen the folder in container
+
+#### Troubleshooting:
+
+Unable to execute git push or pull in container: https://superuser.com/questions/1726204/get-agent-identities-ssh-agent-bind-hostkey-communication-with-agent-failed
+
+### Mac
+
+TODO (@Tzanccc): how to do this on mac
+#### Tools:
+
+* [git](https://git-scm.com/)
+* [VS Code](https://code.visualstudio.com/)
+* [XQuartz](https://www.xquartz.org/)
+* [Docker Desktop](https://www.docker.com/products/docker-desktop/)
+
+#### Instructions:
+
+#### Troubleshooting:
```

Comment:
At least 'Instructions' needs to be filled in.  'Troubleshooting' may be optional.